plugins/modules/zos_job_submit.py

```diff
@@ -124,7 +124,8 @@
 RETURN = r"""
 jobs:
   description:
-     List of jobs output.
+     List of jobs output, or a single NOTFOUND record.
```

Comment:
See my other comment , it appears this doc also needs to be updated, similar to my other comment.

Reply:
I've tweaked it